category.php

```diff
@@ -12,9 +12,10 @@
 
 get_header(); ?>
 
-	<section id="primary" class="site-content">
-		<div id="content" role="main">
+	<div id="stage" class="inner" role="main">
```

Comment:
Is there a reason we are moving off of HTML5 structures like `section` in favor of `div` ?